tests/integration/test_ses.py

```diff
@@ -109,9 +110,23 @@ def test_send_email_can_retrospect(self, ses_client):
         emails_url = config.get_edge_url() + INTERNAL_RESOURCE_PATH + EMAILS_ENDPOINT
         api_contents = requests.get(emails_url).json()
         api_contents = {msg["Id"]: msg for msg in api_contents["messages"]}
+        assert len(api_contents) == 1
         assert message_id in api_contents
         assert api_contents[message_id] == contents
 
+        # Ensure messages can be filtered by email source
+        emails_url = (
+            config.get_edge_url()
+            + INTERNAL_RESOURCE_PATH
+            + EMAILS_ENDPOINT
+            + "?email=none@example.com"
+        )
+        assert len(requests.get(emails_url).json()["messages"]) == 0
+        emails_url = (
+            config.get_edge_url() + INTERNAL_RESOURCE_PATH + EMAILS_ENDPOINT + f"?email={email}"
```

Comment:
To make the match a bit more specific, we could define `email` in line 76 as something like:
```
email = f"user-{short_uid()}@example.com"
```

Reply:
Thanks Waldemar, fixed in [714e80d](https://github.com/localstack/localstack/pull/6340/commits/714e80d1c8cbac449f61048debd22dca834cf24e)